qcodes/instrument_drivers/tektronix/Keithley_2600_channels.py

```diff
@@ -236,19 +250,26 @@ class _MeasurementCurrentParameter(_ParameterWithStatus):
     def __init__(self, *args: Any, **kwargs: Any):
         super().__init__(*args, **kwargs)
 
-    @staticmethod
-    def _parse_response(data: str) -> Tuple[float, MeasurementStatus]:
+    def get_raw(self) -> ParamRawDataType:
+        assert isinstance(self.instrument, KeithleyChannel)
+        assert isinstance(self.root_instrument, Keithley_2600)
 
-        value, meas_status = data.split('\t')
+        smu = self.instrument
+        channel = self.instrument.channel
 
-        status_bits = [int(i) for i in bin(
-            int(float(meas_status))
-        ).replace('0b', '').zfill(16)[::-1]]
+        data = smu.ask(f'{channel}.measure.i(), '
+                       f'status.measurement.instrument.{channel}.condition')
+        value, status = self._parse_response(data)
 
-        if status_bits[1]:
-            return float(value), MeasurementStatus.COMPLIANCE_ERROR
-        else:
-            return float(value), MeasurementStatus.NORMAL
+        self._measurement_status = status
+
+        return value
+
+
+class _MeasurementVoltageParameter(_ParameterWithStatus):
+
+    def __init__(self, *args: Any, **kwargs: Any):
+        super().__init__(*args, **kwargs)
```

Comment:
Since this init method just forwards all arguments to the super class I think we can remove it

Reply:
The same is true for the _MeasurementCurrentParameter class but I guess we missed that in the review of that class